### ISSUE TITLE
org-cite links do not appear in roam buffer

### ISSUE THREAD

**Issue by sopoforic** [org-roam/org-roam#1851]
### Description

After converting my citation links from org-ref links to org-cite links, cite backlinks do not appear in the org-roam buffer.

org-cite is working. On export, the citations are formatted as expected and the bibliography is generated correctly.

#### Steps to Reproduce

1. Have a node **A** with an org-cite ref, e.g. `:ROAM_REFS: @Lukowski2011`
2. Cite that node elsewhere (**B**) with an org-cite link, e.g. `blah blah [cite:@Lukowski2011]`.
3. Turn on the roam buffer with `org-roam-buffer-toggle` in **A**.

#### Expected Results

The ref from **B** should show up in the org-roam buffer for node **A**.

#### Actual Results

cite links do not appear.

These used to work for me with org-ref type links.

### Environment

- Framework: N/A
- Org: Org mode version 9.4.6 (release_9.4.6-642-ga246a1 @ /home/tracy/.emacs.d/hosts/archimedes.barbanon.org/straight/build/org/)
- Org-roam: v2.1.0-16-g1795039
- Emacs: GNU Emacs 28.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.25, cairo version 1.16.0)
 of 2021-07-22
- Org-roam commit: https://github.com/jethrokuan/org-roam/commit/1795039ab93ef19611dbb3fca21c4211c4e655a9

**Comment by chuxubank** [comment]
> Org: Org mode version 9.4.6 (release_9.4.6-642-ga246a1 @/home/tracy/.emacs.d/hosts/archimedes.barbanon.org/straight/build/org/)

You need to use Org 9.5 to get `org-cite` support in `oc.el`, and apply this fix #1849

**Comment by sopoforic** [comment]
I'm using straight to build org from git, so I do have `org-cite` support (it's working fine, as I mentioned above).

Applying the fix linked does make the refs show up (after running `org-roam-db-clear-all` and `org-roam-db-sync`) but they don't have any content. So, I have a ref on the node "Designing the Star User Interface":

```
blah blah blah [cite:@Norman1988].
```

In the roam buffer for Norman1988 I get:

```
Reflinks:
Designing the Star User Interface (Top)
```

There's no content displayed, and I can't visit the node. Pressing enter on it gives `org-roam-buffer-visit-thing: There is no thing at point that could be visited`.

**Comment by chuxubank** [comment]
@sopoforic I forget to mention... You also need `(require 'ox)` may be just after `org-roam` to fix this issue.
The function `org-export-derived-backend-p` is defined in `ox.el`.

**Comment by sopoforic** [comment]
I did `(require 'ox)` and refreshed everything, but the issue persists.

**Comment by chuxubank** [comment]
Then I have no idea...

Mine's just works fine, maybe you need `org-roam-bibtex` package?
Here is my config: [+biblio.el](https://github.com/chuxubank/cat-emacs/blob/main/cats/%2Bbiblio.el) and [+org-roam.el](https://github.com/chuxubank/cat-emacs/blob/main/cats/%2Borg-roam.el)

<img width="1040" alt="image" src="https://user-images.githubusercontent.com/17001575/135180792-20fa8a95-a322-4054-96b6-12d81c875883.png">

**Comment by bdarcus** [comment]
FWIW, I do not have orb installed, but org-cite reflinks work correctly for me.

**Comment by jethrokuan** [comment]
This should be fixed post #1849  for most org versions, so I'm closing this issue. feel free to reopen if it still persists.
